webpack/assets/javascripts/react_app/components/PermissionDenied/PermissionDenied.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+
+const PermissionDenied = ({ missingPermissions, texts, backHref }) => {
+  const { notAuthorizedMsg, permissionDeniedMsg, pleaseRequestMsg } = texts;
+  return (
+    <div className="blank-slate-pf">
+      <div className="blank-slate-pf-icon">
+        <Icon name="lock" color="#9c9c9c" size="2x" title="lock" />
+      </div>
+      {__(notAuthorizedMsg)}
+
+      <h1>{__(permissionDeniedMsg)}</h1>
+      <p>
+        {__(notAuthorizedMsg)}
+        <br />
+        {__(pleaseRequestMsg)}
+        <br />
+      </p>
+      <ul className="list-unstyled">
+        {missingPermissions.map(permission => (
+          <li key={permission}>
+            <strong>{permission}</strong>
+          </li>
+        ))}
+      </ul>
+      <p>
+        <a href={backHref}>
+          <Button>{__('Back')}</Button>
+        </a>
+      </p>
+    </div>
+  );
+};
+
+PermissionDenied.propTypes = {
+  missingPermissions: PropTypes.node,
+  texts: PropTypes.shape({
+    notAuthorizedMsg: PropTypes.string,
+    permissionDeniedMsg: PropTypes.string,
+    pleaseRequestMsg: PropTypes.string,
+  }),
+  backHref: PropTypes.string,
+};
+
+PermissionDenied.defaultProps = {
+  missingPermissions: ['unknown'],
+  texts: {
+    notAuthorizedMsg: 'You are not authorized to perform this action.',
```

Comment:
can you translate the strings here? AFAIK `__()`  should receive a string, not a var.